LaraExtraApi/src-lara/lara/MathExtra.lara

```diff
@@ -1,59 +0,0 @@
-/**
- * @class
- */
-var MathExtra = {};
-
-/**
- * Attempts to simplify a mathematical expression.
- *
- * @param {string|$jp} expression - The expression to simplify.
- * @param {object} constants - An object that maps variable names to constants.
- *
- * @returns {string} simplified expression
- */
-MathExtra.simplify = function(expression, constants) {
-	if(isJoinPoint(expression)) {
-		expression = expression.code;
-	}
```

Comment:
That is true. The move was done since Clava was the only Weaver that actually supported this API, I assumed it was the only one where it would be used.

As for adding `.code` as an attribute of the base joinpoint type (`LaraJoinPoint`), I agree. I had planned for us to have a discussion about this in the near future.